packages/terra-functional-testing/package.json

```diff
@@ -64,13 +64,13 @@
     "platform": "^1.3.6",
     "uuid": "^3.0.0",
     "webdriverio": "^6.5.0",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0",
```

Comment:
I don't think it makes sense to make this one a peer? And I'd hate to add a ton of peer dependencies.